openquakeplatform/test/versions_test.py

```diff
@@ -21,6 +21,11 @@ def page_app_versions_test(self):
         # macthing if the version of applications
         # content numbers and points
         search_version_app = pla.xpath_finduniq(
-            "//table[@class='versions_table']//tr/td[../td[normalize-space(text())='IPT']][normalize-space(text())!='IPT']")
+            "//table[@class='versions_table']//tr/"
+            "td[../td[normalize-space(text())"
+            "='IPT']][normalize-space(text())!='IPT']")
         vers = search_version_app.text
-        self.assertRegexpMatches(vers, '^[0-9]+\.', msg=("version [%s] not valid for IPT app" % vers))
+        self.assertRegexpMatches(vers, '^[0-9]+\.', msg=("version [%s]"
```

Comment:
Please, add (a real) newline before `"version` and concatenate the rest of the string.